src/EventStore.Core/Services/Transport/Grpc/Enumerators.AllSubscriptionFiltered.cs

```diff
@@ -289,7 +289,9 @@ await foreach (var @event in liveEvents.Reader.ReadAllAsync(_cancellationToken)
 							if (startPosition != Position.End) {
 								ReadHistoricalEvents(startPosition);
 							} else {
-								NotifyCaughtUp(startPosition);
+								NotifyCaughtUp(startPosition == Position.End
```

Comment:
`startPosition == Position.End` is necessarily true here because we are in the else?